geomesa-kafka/geomesa-kafka-datastore/pom.xml

```diff
@@ -70,6 +79,13 @@
             <artifactId>kafka_2.10</artifactId>
         </dependency>
         <dependency>
+            <!--This is a compile dependency of Kafka and also KafkaGeoMessageConsoleConsumer -->
+            <groupId>net.sf.jopt-simple</groupId>
+            <artifactId>jopt-simple</artifactId>
+            <version>3.2</version>
+            <scope>compile</scope>
```

Comment:
Storm is using a 3rd version of guava.  What a mess!  I put the guava version back as it was however it seems to me that using different version of guava within different modules of geomesa is not good.  geomesa-kafka-geoserver-plugin is shading and relocating it so external requirements should not affect what version we use but I'll leave this cleanup for another day.